tests/cypress/e2e/create-user.cy.js

```diff
@@ -0,0 +1,55 @@
+describe('Vacation requests', () => {
+
+  before(() => {
+    cy.refreshDatabase()
+    cy.seed('DemoSeeder')
+
+    cy.php(`Toby\\Eloquent\\Models\\User::first();`)
+      .then(user => {
+        cy.login({ email: user.email })
+      })
+  })
+
+  it('Creates a sick vacation request by administrator for another employee', () => {
```

Comment:
Wrong title for this test - this test talks about creating user, not creating sick vacation.